### PR TITLE
[cisa_kev] Update readme with enrich and ES|QL sample, make GA

### PR DESCRIPTION
**Pull request by nicpenning**
- Enhancement

Updated readme with interval, use case sample and made GA.

Please explain:

- WHAT: Update Readme, make GA
- WHY:  The integration has been running long enough that no issue have been reported so making it GA will give it more visibility for use.

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Screenshots

Add this to the readme for a sample use case:
image

### CODE REVIEW

**Comment by nicpenning** [comment]
Ready for review.

File: packages/cisa_kevs/_dev/build/docs/README.md [hunk 1]
```diff
@@ -1,15 +1,47 @@
 # CISA KEV integration
 
-This integration is for [CISA KEV](https://www.cisa.gov/known-exploited-vulnerabilities-catalog) logs. This data can be useful for current awareness of Known Exploited Vulnerabilities according to CISA and also for enriching other vulnerability scan data in the Elastic stack. It includes the following datasets for retrieving logs from the CISA KEV website:
+This integration is for [CISA KEV](https://www.cisa.gov/known-exploited-vulnerabilities-catalog) logs. This data can be useful for current awareness of Known Exploited Vulnerabilities according to CISA and also for enriching other vulnerability scan data in the Elastic stack. The integration checks every 60 minutes by default for the latest CISA KEV list but this can be configured. It includes the following datasets for retrieving logs from the CISA KEV website:
```
Comment (efd6) [inline packages/cisa_kevs/_dev/build/docs/README.md:3]:
```suggestion
This integration is for [CISA KEV](https://www.cisa.gov/known-exploited-vulnerabilities-catalog) logs. This data can be useful for current awareness of Known Exploited Vulnerabilities according to CISA and also for enriching other vulnerability scan data in the Elastic stack. The integration periodically checks for the latest CISA KEV list. It includes the following datasets for retrieving logs from the CISA KEV website:
```
The broad detail is fine for the intro para, the user will see that it's configurable in the UI.

File: packages/cisa_kevs/_dev/build/docs/README.md [hunk 2]
```diff
@@ -1,15 +1,47 @@
...
+{
+  "match": {
+    "indices": ".ds-logs-cisa_kevs.vulnerability-*",
+    "match_field": "vulnerability.id",
+    "enrich_fields": ["cisa_kev.vulnerability.date_added", "cisa_kev.vulnerability.due_date", "cisa_kev.vulnerability.known_ransomware_campaign_use", "cisa_kev.vulnerability.name", "cisa_kev.vulnerability.notes","cisa_kev.vulnerability.product","cisa_kev.vulnerability.required_action","cisa_kev.vulnerability.vendor_project"]
+  }
+}
+```
+
+Here is an example ES|QL query that uses the index pattern of logs-nessus.vulnerability* to enrich the data source with CISA KEV information and keeping the top 10 results. Note, the enrich policy must be created first which has been provided above:
```
Comment (efd6) [inline packages/cisa_kevs/_dev/build/docs/README.md:24]:
```suggestion
Here is an example ES|QL query that uses the index pattern of logs-nessus.vulnerability* to enrich the data source with CISA KEV information and keeping the top 10 results. Note, the enrich policy (shown above) must be created first:
```

File: packages/cisa_kevs/_dev/build/docs/README.md [hunk 3]
```diff
@@ -1,15 +1,47 @@
...
+| limit 10
+```
+
 ## Logs
 
 ### Vulnerabilities
 
 The CISA KEV data_stream retrieves vulnerability information from the endpoint `https://www.cisa.gov/sites/default/files/feeds/known_exploited_vulnerabilities.json`.
 
+
```
Comment (efd6) [inline packages/cisa_kevs/_dev/build/docs/README.md:44]:
```suggestion
```

**Comment by nicpenning** [comment]
@efd6 Any other changes needed here?

**Review by efd6 — APPROVED** [review]

**Comment by nicpenning** [comment]
Thank you for your review!

**Pull request merged**